### ISSUE TITLE
[A11Y] Navbar Menu: States/Properties: The element has missing or incorrect states or properties

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1791]
## Issue Details

The element has missing or incorrect states or properties that are necessary for screen reader users to interact with or understand the content conveyed by the element.
Element description: aria-expanded state for the role="menu".
Element location: Header -> Expand the "Settings" menu button.

---

### Short Description

The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

### Impact

- [ ] Blocker
- [ ] Critical
- [ ] Serious
- [x] Moderate
- [ ] Minor

### Steps To Reproduce:

Expand the navigation menu in Cauldron documentation.

### Current Behavior:

`aria-expanded` is set incorrectly on an element.

### Expected Behavior:

`aria-expanded` is should only be set for the expanding control.

---

### Auditor ID / URL

https://axeauditor.dequecloud.com/test-run/8b259ada-639e-11ef-962d-4f4fe35f0f31/issue/ddd4a57e-8af8-11ef-8a34-bf0f49e9e8b1?sortField=ordinal&sortDir=asc&filter%5Bseverity%5D=2&filter%5Btype%5D=issue&row=0

### Additional Details

N/A

<details>
<summary>Expand for extra context</summary>

- [x] Discovered during VPAT
- [ ] Found using NVDA
- [ ] Found using Chrome Screen Reader
- [ ] Found with special configuration 
</details>

---

## Screenshots 

---

### 1. Perceivable

<details> 
<summary>Expand</summary>

#### 1.1 Text Alternatives

- [ ] 1.1.1 Non-text Content

#### 1.2 Time-based Media

- [ ] 1.2.1 Audio-only and Video-only (Prerecorded)
- [ ] 1.2.2 Captions (Prerecorded)
- [ ] 1.2.3 Audio Description or Media Alternative (Prerecorded)
- [ ] 1.2.4 Captions (Live)
- [ ] 1.2.5 Audio Description (Prerecorded)

#### 1.3 Info and Relationships

- [ ] 1.3.1 Info and Relationships
- [ ] 1.3.2 Meaningful Sequence
- [ ] 1.3.3 Sensory Characteristics
- [ ] 1.3.4 Orientation
- [ ] 1.3.5 Identify Input Purpose

#### 1.4 Distinguishable

- [ ] 1.4.1 Use of Color
- [ ] 1.4.2 Audio Control
- [ ] 1.4.3 Contrast (Minimum)
- [ ] 1.4.4 Resize text
- [ ] 1.4.5 Images of Text
- [ ] 1.4.10 Reflow
- [ ] 1.4.11 Non-text Contrast
- [ ] 1.4.12 Text Spacing
- [ ] 1.4.13 Content on Hover or Focus
</details>

### 2. Operable

<details> 
<summary>Expand</summary>

#### 2.1 Keyboard Accessible

- [ ] 2.1.1 Keyboard
- [ ] 2.1.2 No Keyboard Trap
- [ ] 2.1.4 Character Key Shortcuts

#### 2.2 Enough Time

- [ ] 2.2.1 Timing Adjustable
- [ ] 2.2.2 Pause, Stop, Hide

#### 2.3 Seizures and Physical Reactions

- [ ] 2.3.1 Three Flashes or Below Threshold

#### 2.4 Navigable

- [ ] 2.4.1 Bypass Blocks
- [ ] 2.4.2 Page Titled
- [ ] 2.4.3 Focus Order
- [ ] 2.4.4 Link Purpose (In Context)
- [ ] 2.4.5 Multiple Ways
- [ ] 2.4.6 Headings and Labels
- [ ] 2.4.7 Focus Visible

#### 2.5 Input Modalities

- [ ] 2.5.1 Pointer Gestures
- [ ] 2.5.2 Pointer Cancellation
- [ ] 2.5.3 Label in Name
- [ ] 2.5.4 Motion Actuation
</details>

### 3. Understandable

<details> 
<summary>Expand</summary>

#### 3.1 Readable

- [ ] 3.1.1 Language of Page
- [ ] 3.1.2 Language of Parts

#### 3.2 Predictable

- [ ] 3.2.1 On Focus
- [ ] 3.2.2 On Input
- [ ] 3.2.3 Consistent Navigation
- [ ] 3.2.4 Consistent Identification

#### 3.3 Input Assistance

- [ ] 3.3.1 Error Identification
- [ ] 3.3.2 Labels or Instructions
- [ ] 3.3.3 Error Suggestion
- [ ] 3.3.4 Error Prevention (Legal, Financial, Data)
</details>

### 4. Robust

<details>
<summary>Expand</summary>

#### 4.1 Compatible

- [ ] 4.1.1 Parsing
- [x] 4.1.2 Name, Role, Value
- [ ] 4.1.3 Status Messages
</details>
